### ISSUE TITLE
Missing closing } for vertex_vectors for mesh2 format

### ISSUE THREAD

**Issue by tobltobs** [rsmith-nl/stltools#1]
stl2pov:
If I specify -2 for a mesh2 format, the resulting .inc file can not be opened by povray. It seems that it is lacking a closing `}` for `vertex_vectors`.
Snippet from an example output:
```
...
<0.433883160353, 1.91425704956, 1.0159252882>
  face_indices {
    1018592,
 <0, 1, 2>,
...
```

**Comment by tobltobs** [comment]
If I change in stl2pov.py at line 73 ( more or less)
`lines += ['  face_indices {', '    {},'.format(len(ifacets))]`
to
`lines += ['}  face_indices {', '    {},'.format(len(ifacets))]`
it seems to work. At least povrays parsing doesn't throw an error and is faster. But rendering takes ages and result is gibberish for me.

**Comment by rsmith-nl** [comment]
Fixed in the develop branch at e9986441. Thanks for reporting.
